async_timeout/__init__.py

```diff
@@ -25,12 +25,12 @@ class timeout:
     loop - asyncio compatible event loop
     """
     def __init__(self, timeout: Optional[float],
-                 *, loop: asyncio.AbstractEventLoop=None) -> None:
+                 *, loop: Optional[asyncio.AbstractEventLoop] = None) -> None:
```

Comment:
Hmm, that should not be necessary. Per PEP 484:

> An optional type is also automatically assumed when the default value is None, for example:

Reply:
In python/typing#275 we agreed to deprecate that part of PEP 484, but it looks like the PEP hasn't actually been changed yet.